js-modules/prettify.js

```diff
@@ -57,9 +57,9 @@
 // JSLint declarations
 /*global console, document, navigator, setTimeout, window, define */
 
-include("defs.js");
-
-var HACK_TO_FIX_JS_INCLUDE_PL;
+/* @ifndef RUN_PRETTIFY */
```

Comment:
From what I understand this is a limitation of closure compiler: https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#typedef-type, so I tried to keep things the same as what `js_include.pl` was doing, so that `defs.js` typedefs are always included top-level in both `prettify.js` and `run_prettify.js`